imsim/stamp.py

```diff
@@ -84,6 +84,27 @@ def setup(self, config, base, xsize, ysize, ignore, logger):
         Returns:
             xsize, ysize, image_pos, world_pos
         """
+
+        # First do a parsing check to make sure that all the options passed to
+        # the config are valid, and no required options are missing.
+
+        req = {'fft_sb_thresh': float,
+               'band': str,
```

Comment:
This is optional (sort of).  It's only required for the same code path that hits rawSeeing and airmass.  I.e. simulating moderately bright stars (flux>10) if stamp.size is not given explicitly.